pages/hub/apps/guides/azuro-score.mdx

```diff
@@ -1,203 +1,10 @@
-# Get Azuro Score leaderboard
+import { Callout } from 'components'
 
-Follow this guide to fetch data required to show Azuro Score leaderboard:
-- [Total](#fetch-total-leaderboard)
-- [Total including specific bettor](#fetch-total-leaderboard-including-specific-bettor)
-- [Increment by specific period](#fetch-leaderboard-for-specific-period)
-- [Increment by specific period including specific bettor](#fetch-leaderboard-for-specific-period-including-specific-bettor)
+# Get Azuro Score leaderboard
 
 If you not familiar with base concept about Azuro Score, read it [here](/concepts/azuro-score/overview).
```

Comment:
better remove this as well

Reply:
https://github.com/Azuro-protocol/gem-docs/pull/199/commits/6712abdfb5d5239837b7ffb1b11523d775b8c66d